var/spack/repos/builtin/packages/py-meson-python/package.py

```diff
@@ -14,17 +14,21 @@ class PyMesonPython(PythonPackage):
 
     maintainers = ["eli-schwartz", "adamjstewart"]
 
+    version("0.12.0", sha256="8cb159a8093a2e73cfa897f8092ec93b74e3842f94dff7fde381c6fe0e0b064d")
+    version("0.11.0", sha256="110258837c2ffe762f5f855c7ea5385f1edd44074e93a0f317ffefc7aab42b09")
```

Comment:
I'm a little confused why this is showing up here. It's a subset of what was merged in gh-34675

Reply:
Oh, my branch is out of date, let me rebase